src/components/hooks/graphql/useTasksQuery.ts

```diff
@@ -0,0 +1,74 @@
+import { gql, useQuery } from '@apollo/client';
+
+export const GET_ALL_TASKS = gql`
+  query allTasks {
+    allTasks(condition: { isDeleted: false }) {
+      edges {
```

Comment:
i dont remember but can't you use nodes instead of edges and make the flow easier?

Reply:
good idea, thanks (: